test/resend_serialize_ptr_size.cpp

```diff
@@ -12,6 +12,27 @@ BOOST_AUTO_TEST_SUITE(test_resend_serialize_ptr_size)
 
 using namespace MQTT_NS::literals;
 
+template <typename Client>
+inline void connect_no_clean(Client& c) {
```

Comment:
This looks like the same code in multiple files.

Can it be put in a header file?

Reply:
Yes I will move it to header file.
I've checked existing header files but not found appropriate one. So I will create `test_util.hpp`.